MapboxDirectionsCLI.md

```diff
@@ -0,0 +1,41 @@
+#  Mapbox Directions CLI
+
+## Getting Started
+`MapboxDirectionsCLI` is a command line tool, designed to round-trip an arbitrary, JSON-formatted Directions or Map Matching API response through model objects and back to JSON. This is useful for various scenarios including testing purposes and designing more sophisticated API response processing pipelines. It is supplied as a Swift package.
```

Comment:
“MapboxDirectionsCLI” isn’t a name we seem to be using anywhere from a user perspective.

```suggestion
`mapbox-directions-swift` is a command line tool, designed to round-trip an arbitrary, JSON-formatted Directions or Map Matching API response through model objects and back to JSON. This is useful for various scenarios including testing purposes and designing more sophisticated API response processing pipelines. It is supplied as a Swift package.
```

By the way, this description will need to be revisited as part of #576, which makes the tool into a more straightforward command-line version of normal usage of the library.

Reply:
I'll add a note to that pr as a reminder.